doc/developers-guide/accumulators.md

```diff
@@ -3,25 +3,25 @@
 XMPP stanza processing starts in the `ejabberd_c2s` module, which receives the stanza from a socket, or in `ejabberd_s2s_in` which receives stanzas from federated XMPP clusters.
 The stanza is processed and eventually it and/or other messages are sent out, either to the original sender, to another c2s process within the same MongooseIM installation, or to another XMPP server.
 
-At the beginning of the main processing chain an accumulator is created containing the original stanza, some values extracted from the stanza and additional info:
-
-* ref - a unique reference of the acc, useful for tracing
-* timestamp - current time
-* element - the original stanza
-* name - <<"message">>, <<"presence">> or <<"iq">>
-* type - e.g. <<"set">> (in iq)
-* attrs - attributs of the root xml element of the stanza
-* from, to - full jids of the sender and recipient in binary form
-* from_jid, to_jid - same as #jid{} records.
-* origin_sid - session id (SID) of `ejabberd_c2s`. Set when we receive a stanza
-  from the user device. *property*, *optional*.
-* origin_jid - full JID of `ejabberd_c2s`.
-  Set the same time as `origin_sid`. *property*, *optional*.
+At the beginning of the main processing chain an accumulator is created containing following set of keys:
+
+* `ref` - A unique reference of the acc, useful for tracing.
+* `timestamp` - An Erlang timestamp retrieved from `os:timestamp()`.
+* `origin_pid` - A PID of the process that created the accumulator.
+* `origin_location` - `{Module, Function Line}` - A place in the code where accumulator was created.
```

Comment:
`origin_location` - `{Module, Function Line}` - A place in the code where the accumulator was created.